common/zhash.c

```diff
@@ -540,10 +540,11 @@ uint32_t zhash_str_hash(const void *_a)
 
     char *a = * (char**)_a;
 
-    int32_t hash = 0;
+    uint32_t hash = 0;
```

Comment:
In this case, we can skip the `(uint32_t) hash` cast later.

Reply:
oops, ty